### PR TITLE
Fix two typos in EIP-1812.md

### PR DESCRIPTION
**Pull request by ppoliani**
Fix minor typos

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1812 requires approval from one of (@pelle)

**Comment by axic** [comment]
@pelle can you please review?

**Comment by axic** [comment]
Ping @pelle

File: EIPS/eip-1812.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ require(!revocations.revoked(claim.subject, digest), "claim was revoked by subje
 
 ### Creation of Verifiable Claims Domains
 
-Creating specific is Verifiable Claims Domains is out of the scope of this EIP.   The Example Code has a few examples.
+Creating specific Verifiable Claims Domains is out of the scope of this EIP.   The Example Code has a few examples.
```
Comment (pelle) [inline EIPS/eip-1812.md]: Thanks for this, this is definitely a typo

File: EIPS/eip-1812.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ A Name Claim
 struct Name {
 	address issuer;
 	address subject;
-	string name;
+	string value;
```
Comment (pelle) [inline EIPS/eip-1812.md]: this was intentional to show semantic meaning naming being possible

Comment (axic) [reply EIPS/eip-1812.md:245]: @ppoliani I think this answer is for you

**Pull request closed without merging**